README.md

```diff
@@ -64,9 +64,54 @@ $ cargo test --release
 A basic [example](https://github.com/poanetwork/hbbft/blob/master/examples/README.md) is included to run a network simulation.
 
 ```
+$ cargo run --example simulation --release
+```
+
+![Screenshot](screenshot.png)
+
+|  Heading    | Definition                       
+| ----------- | -------------------------------------------------------------------------- | 
+| Epoch       |  Epoch number. In each epoch, transactions are processed in a batch by simulated nodes (default is 10 nodes) on a network.                                                            | 
+| Min Time   | Timestamp on the _first_ Tx (transaction) processed in a batch of transactions.                                         |  
+| Max Time   | Timestamp on the _last_ Tx in a batch.                                                  |  
+| Txs         | Number of transactions processed in the epoch.                                           |  
+| Msgs/Node   | Average number of messages handled by a node. The counter is cumulative and includes the number of messages handled in the current epoch and all previous epochs.                                                               |  
+| Size/Node   | Average message size (in kilobytes) handled by a node. This is cumulative and includes message size for the current epoch and all previous epochs.                                                             |  
```

Comment:
It uses "MB" (or "GB"), too, if the numbers get large enough, so it's not always in kilobytes.